content/articles/domains-be.markdown

```diff
@@ -16,7 +16,7 @@ This article explains the requirements and special procedures for .BE domain nam
 
 ## Changing the Contact for a .BE domain {#trade}
 
-We don't currently support changing the registrant. Please contact us at [support@dnsimple.com](mailto:support@dnsimple.com), or through the [contact page](https://dnsimple.com/contact) for further assistance.
+Changing the registrant requires a `TRADE` operation with the registry and the payment of an additional fee. Please contact us at [support@dnsimple.com](mailto:support@dnsimple.com), or through the [contact page](https://dnsimple.com/contact) for further assistance.
```

Comment:
While this is technically correct, do we expose those internal mechanisms/naming? We do not call it such within the app for example :)

Reply:
Fair point—that info may be better suited for the wiki along with the exact steps needed.